### ISSUE TITLE
Fix Failed Workflow

### ISSUE THREAD

**Issue by camerondurham** [camerondurham/cs350-docker#7]
**Describe the bug**

Failed workflow building and publishing image: https://github.com/camerondurham/cs350-docker/runs/3476950995?check_suite_focus=true

**To Reproduce**
1. Build Docker image

Docker build fails with following error, need to run in Ubuntu environment to reproduce:

```
Error: buildx failed with: error: failed to solve: process "/bin/bash -o pipefail -c apt-get update && apt-get install -y --no-install-recommends     apt-utils=1.6.13     build-essential=12.4ubuntu1     cmake=3.10.2-1ubuntu2.18.04.1     g++=4:7.4.0-1ubuntu2.3     gcc=4:7.4.0-1ubuntu2.3     gcc-multilib=4:7.4.0-1ubuntu2.3     gdb=8.1.1-0ubuntu1     gdbserver=8.1.1-0ubuntu1     openssh-server=1:7.6p1-4ubuntu0.3     qemu=1:2.11+dfsg-1ubuntu7.36     qemu-system-common=1:2.11+dfsg-1ubuntu7.36     rsync=3.1.2-2.1ubuntu1.1     && apt-get autoclean -y     && apt-get autoremove -y     && rm -rf /var/lib/apt/lists/*     && mkdir /var/run/sshd     && echo 'root:root' | chpasswd     && sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config     && sed 's@session\\s*required\\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd" did not complete successfully: exit code: 100
```

**Comment by camerondurham** [comment]
Probably due to failed `apt-get` command and pinned the pinned versions being used.

This SO might be helpful: https://stackoverflow.com/a/38004106/4676641

**Comment by camerondurham** [comment]
Closed by 5770faf
